_maps/map_files/templates/dungeons/north_sewer_2.dmm

```diff
@@ -196,7 +196,10 @@
 	id = "hell";
 	name = "Door Control"
 	},
-/turf/closed/wall/r_wall/rust,
+/turf/closed/wall/r_wall/rust{
```

Comment:
Lets refrain from editing vars on objects in StrongDMM and add them as a proper subtype in code.

Reply:
Added a new indestructible ver of the reinforced wall instead to fix this.